### PR TITLE
Place to add (and test) quick features

### PR DESCRIPTION
**Pull request by jmsull**

### CODE REVIEW

**Comment by jmsull** [comment]
I added the E polzn source function and some functions to compute clte, clee (which addesses #63) - all untested! Also addressed #62 by lazily copying and pasting the primordial power lines I used in the Plin function.

File: src/perturbations.jl [hunk 1]
```diff
@@ -384,3 +384,24 @@ function source_function(du, u, hierarchy::Hierarchy{T, BasicNewtonian}, x) wher
...
+    Tν =  (par.N_ν/3)^(1/4) *(4/11)^(1/3) * (15/ π^2 *ρ_crit(par) *par.Ω_r)^(1/4)
+    Ω_ν =  7*(2/3)*par.N_ν/8 *(4/11)^(4/3) *par.Ω_r
+    logqmin,logqmax=log10(Tν/30),log10(Tν*30)
+    q_pts = xq2q.(bg.quad_pts,logqmin,logqmax)
+    Θ, Θᵖ, 𝒩, ℳ, Φ, δ, v, δ_b, v_b = unpack(u, hierarchy)  # the Θ, Θᵖ are mutable views (see unpack)
+    Θ′, Θᵖ′, 𝒩′, ℳ′, Φ′, δ′, v′, δ_b′, v_b′ = unpack(du, hierarchy)
+
+
+    Π = Θ[2] + Θᵖ[2] + Θᵖ[0]
+    return (3/(4k^2)) * g̃ₓ * Π
```
Comment (xzackli) [inline src/perturbations.jl:406]: SZ 12d also includes some higher-order terms (1st and 2nd derivatives of the photon perturbations Π). Skipping for now?

**Comment by jmsull** [comment]
The class comparison test for perturbations failed because it is against a nonu run - will rename the test to be a nonu test where mnu is explicitly set to zero.

**Comment by jmsull** [comment]
This is marked as updated in the manifest...

**Comment by jmsull** [comment]
There is now a collision between "LinearInterpolations" in the 13 version of Interpolations.jl and the current version of DataInterpolations.jl. The solution is to update Interpolations.jl to 14 (which I tried to do above but failed)

**Comment by jmsull** [comment]
We need to merge this so that people can use the main branch - I will look at this again to try to make that happen - even if it means dropping DataInterpolations temporarily.

**Pull request merged**